backend/nlp/handler.py

```diff
@@ -14,11 +32,52 @@ def hello(event, context):
 
     return response
 
-    # Use this code if you don't use the http event with the LAMBDA-PROXY
-    # integration
-    """
-    return {
-        "message": "Go Serverless v1.0! Your function executed successfully!",
-        "event": event
-    }
-    """
+
+
+"""
+"""
+def analyze_text(text, analyzer):
+    sentiment = analyzer.polarity_scores(text)
+    return sentiment
+
+
+
+"""
+"""
+def analyze_reviews(reviews_list):
+    s = SentimentIntensityAnalyzer()    # Initialize the analyzer object
+    list_process_reviews = []           # Initialize the return list
+
+    for index, review in enumerate(reviews_list):
+        good = 0.0
+        bad = 0.0
+        comp = 0.0
+
+        processed_review = review
+
+        if review["text"].strip() != "":
```

Comment:
Compared to the examples at http://www.nltk.org/howto/sentiment, it seems like we're barely using any of the NLTK modules... Is this something we're planning to look into when we implement training? I think we should either revise this code now or, if more research needs to happen, add comments to explain future work.